jujugui/static/gui/src/app/store/env/test-acl.js

```diff
@@ -120,10 +120,10 @@ describe('ACL', function() {
 
   it('canRemoveModel', () => {
     const model = makeModel('who', {
```

Comment:
For completeness you could consider adding a second test where the owner has the domain e.g. `const model = makeModel('who@external', {`